### PR TITLE
Cleaned eslint for webapp

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Removing unused globals and overrides in webapp's eslint.

https://github.com/medic/angular10-migration/issues/117

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: webapp/.eslintrc [hunk 1]
```diff
@@ -5,34 +5,12 @@
     "node": true
   },
   "plugins": ["compat"],
-  "globals": {
-    "angular": true,
-    "Tour": true
```
Comment (latin-panda) [inline webapp/.eslintrc:10]: "angular" global not supported in new framework version. 

"Tour" global is used in places: 
   1. `bootstrap-tour-standalone.js` Currently not covered in the lint evaluation, it'd need several fixes for lint issues, example: double quote to single quote strings. Let me know if to include the file, I think this original code from a lib, so I think it's fine to exclude.
   2. `main.ts` accessed via window object: `window.Tour`. This is fine, no need rule.
   3. `tour.service.ts` accessed via window object. This is fine, no need rule.

File: webapp/.eslintrc [hunk 2]
```diff
@@ -60,7 +38,6 @@
         "@angular-eslint/use-lifecycle-interface": "warn",
         "@angular-eslint/use-pipe-transform-interface": "error",
 
-        // todo remove this after implementing some sort of logging service
```
Comment (latin-panda) [inline webapp/.eslintrc:63]: I added a comment in the respective ticket. If ever picked up.

File: webapp/tests/karma/utils.js [hunk 3]
```diff
@@ -24,14 +24,15 @@ window.KarmaUtils = {
   nullPromise: () => () => Q.defer().promise,
   mockDB: db => () => () => db,
   setupMockStore: function(initialState, mocks = {}) {
+    /* ToDo: migrate and/or clean
```
Comment (latin-panda) [inline webapp/tests/karma/utils.js:27]: This will be cleaned in this ticket: https://github.com/medic/angular10-migration/issues/126

**Comment by latin-panda** [comment]
Hi @dianabarsan  can you please take a look at this one? Thanks

**Review by dianabarsan — APPROVED** [review]
Great! Thanks! 

Also, thanks for taking the time to polish the title and the description of the PR ❤️

**Pull request merged**